db/migrate/20241130150002_add_interesting_repository_updates.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddInterestingRepositoryUpdates < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_column :repositories, :interesting, :boolean
```

Comment:
should this default to `false`?

Reply:
the problem is that it will block the migration while it fills in false for 84M rows